source/includes/_ingress.md

```diff
@@ -1,5 +1,11 @@
 # Ingress API
 
+Any of the endpoints below will return:
+
+- **200** http status if everything was ok.
+- **400** http status if some parameters were invalid (It may specify exactly which ones.
```

Comment:
```suggestion
- **400** http status if some parameters were invalid (It may specify exactly which ones.
+ **400** http status if some parameters were invalid (It may specify exactly which ones.)
```